net/bind/Makefile

```diff
@@ -24,9 +24,6 @@ PKG_SOURCE_URL:= \
 	https://ftp.isc.org/isc/bind9/$(PKG_VERSION)
 PKG_HASH:=4da2d532e668bc21e883f6e6d9d3d81794d9ec60b181530385649a56f46ee17a
 
-PKG_FIXUP:=autoreconf
-PKG_REMOVE_FILES:=aclocal.m4 libtool.m4
```

Comment:
It would be nice to understand why this is suddenly breaking the build, but I can confirm that it builds ok with these lines removed and fails when they're present.

Reply:
I had the impression that the `autoreconf` fixup fails to actually fixup the autotool mess in bind9 (I seem to recall that there's like 8k lines of Makefiles and magic....but it's at least been replaced with meson in the development branch of bind). IIRC there were one or more '.rej' files in the build dir (which would indicate a failed attempt at patching files)